integration-cli/requirements.go

```diff
@@ -74,6 +74,16 @@ var (
 		},
 		fmt.Sprintf("Test requires an environment that can host %s in the same host", v2binary),
 	}
+	NotaryHosting = TestRequirement{
+		func() bool {
+			// for now notary binary is built only if we're running inside
+			// container through `make test`. Figure that out by testing if
+			// registry binary is in PATH.
```

Comment:
fail, I thought I erased the evidence of copying the registry requirement :smile:, it is fixed now